libzeth/circuits/circuits-utils.cpp

```diff
@@ -2,7 +2,7 @@
 //
 // SPDX-License-Identifier: LGPL-3.0+
 
-#include "circuits-utils.hpp"
+#include <libzeth/circuits/circuits-utils.hpp>
```

Comment:
Shall we `circuit-utils.*` to `circuit_utils` for consistency?

Reply:
Let's do that indeed. Good catch